e2e/windows/e2e-helper.sh

```diff
@@ -81,10 +81,26 @@ upload_linux_file_to_storage_account() {
         log "Upload linux file successfully"
     else
         err "Failed to upload linux file. Error code is $retval."
+        exit 1
     fi
 }
 
 download_linux_file_from_storage_account() {
+    local retval
+    retval=0
+    if [[ "$(check_linux_file_exists_in_storage_account)" == *"Linux file already exists in storage account."* ]]; then
```

Comment:
Suggest using `isRemoteCsePackageExist=$(az storage blob exists --auth-mode login --blob-url $cseBlobUrlForUploading | jq -r '.exists')`

We have the code in e2e/windows/e2e-scenario.sh. You can wrap it as a function

Reply:
check_linux_file_exists_in_storage_account is just a function. I have changed the way of checking file existence in this function.